microshift_release_notes/microshift-4-12-release-notes.adoc

```diff
@@ -6,15 +6,69 @@ include::_attributes/attributes-microshift.adoc[]
 
 toc::[]
 
-// Update All for MicroShift; text is FPO (for placement only)
-Red Hat {product-title} provides developers and IT organizations with small form factor and edge computing, delivered as an application that customers can deploy on top of their managed {op-system-first} devices at the edge. Built on {op-system-first} and Kubernetes, {product-title} provides an efficient way to operate single-node clusters in low-resource edge environments.
+{product-title} provides developers and IT organizations with small-form-factor and edge computing, delivered as an application that customers can deploy on top of their managed {op-system-first} devices at the edge. Built on {op-system-first} and Kubernetes, {product-title} provides an efficient way to operate single-node clusters in low-resource edge environments.
 
 {product-title} is designed to make control plane restarts economical and be lifecycle-managed as a single unit by the operating system. Updates, roll-backs, and configuration changes consist of simply staging another version in parallel and then - without relying on a network - flipping to and from that version and restarting.
 
-//[id="microshift-4-12-about-this-release"]
+[id="microshift-4-12-about-this-release"]
 == About this release
 
-// TODO: Update with the relevant information/ links.
-{product-title} is now available as a Developer Preview. Features and known issues that pertain to {product-title} {product-version} are included in this topic.
+{product-title} is now available as a Developer Preview software. Features and known issues that pertain to {product-title} {product-version} are included in this topic. For more information about the support scope of Red Hat Developer Preview software, see link:https://access.redhat.com/support/offerings/devpreview/[Developer Preview Support Scope].
 
-{product-title} {product-version} is supported on {op-system-first} 4.12 and {op-system-base-full} 8.4 and 8.5.
+[IMPORTANT]
+====
+Red Hat will not provide or support an update or upgrade path from Developer Preview and Technology Preview versions to later versions of {product-title}. A new installation will be necessary.
+====
+
+[id="microshift-known-issues"]
+== Known issues
+* There is an issue exporting services via the Route API when the host and routerCanonicalHostname do not include the sub domain name. To work around this problem, use the subdomain setting when creating a Route.
++
+.Example
+[source, terminal]
+----
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: route-1
+spec:
+  subdomain: svc-subdomain  ⇐ specify the subdomain
+  port:
+    targetPort: 8080
+  to:
+    kind: Service
+    name: svc-1
+----
+
+* There is an issue resuming pods when a service is idled. Unidling is not supported by the {product-title} software, and manual re-scaling the replicas is required.
```

Comment:
Unidling [isn't actually a word](https://www.merriam-webster.com/dictionary/unidle), is it? I can't think of a good alternative, though.

Reply:
"Returning a service to an active state" perhaps?

Reply:
We use the term "unidling" elsewhere in the docs. https://access.redhat.com/documentation/en-us/openshift_container_platform/4.1/html/applications/idling-applications#idle-unidling-applications_idling-applications

We should make sure it's clear here that services are not scaled back up automatically.

```suggestion
* There is an issue resuming pods when a service is idled. {product-title} does not support automatically reactivating services, and manual re-scaling the replicas is required.
```

Reply:
Thank you, @mburke5678 and @dhellmann, edit made.